### DISCUSSION TITLE
Space filling overview mode for slides

### DISCUSSION THREAD

**Discussion by mickmcq** [quarto-dev/quarto-cli#7973] [Feature Requests]
### Description

Opening overview mode in revealjs slides uses only a small portion of the screen. A long horizontal line of thumbnails with some vertical columns of thumbnails requires a lot of scrolling. Other slideware creates a grid of thumbnails, using up all the screen space, making it easier to navigate to the desired slide with one click instead of much scrolling followed by a click. As a bonus, other slideware often allows zooming into the "slide sorter tray", showing more detail per thumbnail at the risk of some scrolling becoming necessary.

**Comment by cscheid** [comment]
I think you want to discuss this in the revealjs repository, right? https://github.com/hakimel/reveal.js

**Comment by mcanouil** [comment]
This is a Reveal.js feature request: https://github.com/hakimel/reveal.js
There is nothing here that can be done on Quarto side.

**Comment by mickmcq** [comment]
Sorry for the noise! I will pursue this in revealjs. Thanks for the pointer.
